metriq_gym/benchmarks/mirror_circuits.py

```diff
@@ -49,76 +49,81 @@ class MirrorCircuitsData(BenchmarkData):
 def select_optimal_qubit_subset(topology_graph: rx.PyGraph, target_width: int) -> list[int]:
     """
     Select a connected subset of qubits with the best connectivity.
-    
+
     Args:
         topology_graph: The device connectivity graph.
         target_width: Number of qubits to select.
-        
+
     Returns:
         List of selected qubit indices.
     """
     all_qubits = list(topology_graph.node_indices())
-    
+
     if target_width >= len(all_qubits):
         return all_qubits
-    
+
     if target_width == 1:
         return [all_qubits[0]]
-    
+
     # Use a greedy approach similar to CLOPS create_qubit_map
     # Start from the qubit with highest degree and expand
     degrees = [(node, topology_graph.degree(node)) for node in all_qubits]
     degrees.sort(key=lambda x: x[1], reverse=True)
-    
+
     selected = [degrees[0][0]]  # Start with highest degree node
     remaining = set(all_qubits) - {degrees[0][0]}
-    
+
     # Greedily add connected qubits
     while len(selected) < target_width and remaining:
         best_candidate = None
         max_connections = -1
-        
+
         for candidate in remaining:
             # Count connections to already selected qubits
-            connections = sum(1 for selected_qubit in selected 
-                            if topology_graph.has_edge(candidate, selected_qubit))
+            connections = sum(
+                1
+                for selected_qubit in selected
+                if topology_graph.has_edge(candidate, selected_qubit)
+            )
             if connections > max_connections:
                 max_connections = connections
                 best_candidate = candidate
-        
+
         if best_candidate is not None:
             selected.append(best_candidate)
             remaining.remove(best_candidate)
         else:
             # No more connected qubits, add any remaining
             selected.append(list(remaining)[0])
             remaining.remove(selected[-1])
-    
+
     return selected[:target_width]
 
 
-def create_subgraph_from_qubits(topology_graph: rx.PyGraph, selected_qubits: list[int]) -> rx.PyGraph:
+def create_subgraph_from_qubits(
```

Comment:
Some interesting "word-wrap" and line spacing changes, but maybe that's all okay and consistent with what we want here.

Reply:
It was Ruff precommit hook modifying this file, not me